t/test-lib.sh

```diff
@@ -891,15 +891,7 @@ xml_attr_encode () {
 	# We do not translate CR to &#x0d; because BSD sed does not handle
```

Comment:
Is the CR comment stale now?  See line 56 in test-xml-encode.c

Reply:
Ooops again!